riscv-asm.md

```diff
@@ -426,6 +426,44 @@ msg:
 	    .string "Hello World\n"
 ```
 
+Function Calls
+--------------
+
+The following pseudo instructions are available to call subroutines far from
+the current position:
+
+  * `call	<rd>, <symbol>`: call away subroutine[^1]
```

Comment:
I'd make call symbol and call rd,symbol two separate lines, but I don't think this is important.  The following example shows that rd, can be omitted.